cookbook/navigation/returning-data.md

```diff
@@ -222,45 +220,43 @@ class SelectionButton extends StatelessWidget {
     // Navigator.pop on the Selection Screen!
     final result = await Navigator.push(
       context,
-      new MaterialPageRoute(builder: (context) => new SelectionScreen()),
+      MaterialPageRoute(builder: (context) => SelectionScreen()),
     );
 
     // After the Selection Screen returns a result, show it in a Snackbar!
-    Scaffold
-        .of(context)
-        .showSnackBar(new SnackBar(content: new Text("$result")));
+    Scaffold.of(context).showSnackBar(SnackBar(content: Text("$result")));
```

Comment:
Same question.

Reply:
Yep, formatter on Travis failed on this one as well.